lib/gps_sim/Kconfig

```diff
@@ -103,6 +114,18 @@ config GPS_SIM_TRIGGER_USE_BUTTON
 
 endchoice
 
+config GPS_SIM_GPIO_CONTROLLER
+	string "GPIO controller"
+	default "GPIO_0"
+	help
+		GPIO controller for GPS simulator trigger button.
+
+config GPS_SIM_GPIO_PIN
+	int "GPIO pin"
+	default -1
```

Comment:
It was done to prevent it being set to some arbitrary, legal, value that could potentially interfere with other board functions suvh as UART. Looks weird though, so I can change it to something, but not sure what pin number makes most sense. Will look at it again.